packages/backend/vite.config.js

```diff
@@ -50,6 +50,8 @@ const config = {
       external: [
         '@podman-desktop/api',
         '@huggingface/gguf',
+        'express',
```

Comment:
This turn out not to be working out in production build, as it completely exclude the package and make it not being packed in the output.